src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/LocalSlotMappingTests.cs

```diff
@@ -73,7 +73,7 @@ .maxstack  1
 
             var diff1 = compilation1.EmitDifference(
                 generation0,
-                ImmutableArray.Create(new SemanticEdit(SemanticEditKind.Update, method0, method1, syntaxMap: null, preserveLocalVariables: true)));
+                ImmutableArray.Create(SemanticEdit.Create(SemanticEditKind.Update, method0, method1, syntaxMap: null, preserveLocalVariables: true)));
```

Comment:
>SemanticEdit.Create [](start = 38, length = 19)

Would we need `SemanticEdit.Create` if `GetMember<T>` helper returned a public symbol?

Reply:
> Would we need SemanticEdit.Create if GetMember<T> helper returned a public symbol?

```GetMember<MethodSymbol>``` cannot return IMethodSymbol without changes at the call site. In general, I do not think it is desirable for GetMember to return a public symbol when called on an internal symbol.

---
In reply to: [340385708](https://github.com/dotnet/roslyn/pull/39498#discussion_r340385708) [](ancestors = 340385708)